test/lib/java-wrapper.test.ts

```diff
@@ -26,7 +26,7 @@ test('classes per jar mapping is created', async () => {
 
 test('not target folder throw error', async () => {
   expect(
-    getTargets('some-bogus-folder-that-does-not-exist'),
+    getTargets('some-bogus-folder-that-does-not-exist', 'mvn'),
```

Comment:
Do we have a gradle test?

Reply:
It's not needed for non-existing upper level folder.

Reply:
Meant more if we have things that test gradle?

Reply:
Not sure we understand each other - this is not maven specific test.

In general, coverage is roughly the same.